bc_obps/registration/tests/endpoints/test_useroperator_endpoints.py

```diff
@@ -176,15 +165,14 @@ def test_unauthorized_users_cannot_put(self):
             )
             assert response.status_code == 401
 
-    def test_get_user_operator_status(self):
+    def test_get_user_operator(self):
         user_operator = user_operator_baker()
         user_operator.user_id = self.user.user_guid
         user_operator.save(update_fields=['user_id'])
-        response = TestUtils.mock_get_with_auth_role(
-            self, 'industry_user', f"{base_endpoint}user-operator-status-from-user"
-        )
+        response = TestUtils.mock_get_with_auth_role(self, 'industry_user', f"{base_endpoint}user-operator-from-user")
         assert response.status_code == 200
         assert response.json()['status'] == user_operator.status
+        assert response.json().get('is_new') is not None
```

Comment:
I tested one sample field from the UserOperator ModelField, and is_new because it's from operator